packages/utilities/package.json

```diff
@@ -64,9 +64,11 @@
     "dompurify": "2.0.11",
     "fetch-mock": "^7.3.6",
     "gzip-size": "^5.0.0",
+    "isomorphic-dompurify": "0.4.0",
```

Comment:
Just wondered (my apologies if I was missing anything). Did you mean to put this to `dependencies` instead of `devDependencies`? Probably for `marked` as well.

Reply:
Yes I did, thanks @asudoh!